javalib/src/main/scala/java/io/BufferedOutputStream.scala

```diff
@@ -0,0 +1,80 @@
+package java.io
+
+/**
+ * Created by remi on 08/03/17.
+ */
+class BufferedOutputStream(out: OutputStream, size: Int)
+    extends FilterOutputStream(out)
+    with Flushable
+    with Closeable
+    with AutoCloseable {
+
+  def this(in: OutputStream) = this(in, 8192)
+
+  /** The internal buffer array where the data is stored. */
+  private[this] var buf = new Array[Byte](size)
+
+  /** The number of valid bytes in the buffer. */
+  private[this] var count = 0
+
+  private[this] var closed = false
+
+  /**
+   * Closes this Output stream and releases any system resources associated with the stream.
+   */
+  override def close(): Unit = {
+    flush()
```

Comment:
This will throw an exception if you close the stream more than once. However, the javadoc says:

> If the stream is already closed then invoking this method has no effect.